docs/data/stashcache/install-cache.md

```diff
@@ -18,8 +18,7 @@ Before starting the installation process, consider the following requirements:
 * __Operating system:__ Ensure the host has [a supported operating system](../../release/supported_platforms.md)
 * __User IDs:__ If they do not exist already, the installation will create the Linux user IDs `condor` and
   `xrootd`
-* __Host certificate:__ Required for reporting and authenticated caches.
-  Authenticated caching is an optional feature.
+* __Host certificate:__ Required for contacting the central OSDF redirector.
```

Comment:
I'd hope that a host cert also provides a way for the client to verify they're talking to the expected host!

Reply:
How about I just say that it's required for HTTPS?

Reply:
Well it's also used in the GSI handshake, too, right? Maybe just say it's for authentication?

Reply:
Sure.